modules/core/src/lib/init.js

```diff
@@ -41,7 +41,7 @@ if (existingVersion && existingVersion !== version) {
 
 if (!existingVersion) {
   // eslint-disable-next-line
-  if (process.env.NODE_ENV !== 'production') {
+  if (isBrowserMainThread() && process.env.NODE_ENV !== 'production') {
```

Comment:
Can you just test for `NODE_ENV==='development'` instead?

Reply:
Rather than adding these checks, perhaps we could just change the priority of the log statement to `1`. This means we can always turn it on with probe.gl but it would not shown by default.

Reply:
Happy to make either of those changes. Though setting this log priority to `1` seems to defeat the purpose of the log message since if you already had the log level set to `1` you wouldn't need to see this message.

Reply:
I think I understood your point now. The advantage of still having the message emitted in all cases is that if someone asks me to debug an issue in a deployed deck.gl application, I can go into the console, set the log level, reload the page, and immediately see what version of deck.gl it was built with.

That works well because the probe.gl logger ensuring that log level changes are persisted in local storage across page reloads.